content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,121 +1,78 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+DVC can rely on existing Git servers or hosting (e.g. GitHub, GitLab, etc.) to
+save and share DVC Experiments.
 
-[DVC remotes](/doc/command-reference/remote) on the other hand are
-storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
-configure with `dvc remote`. DVC uses them to store and fetch large files that
-don't normally fit inside Git repos.
+<details>
```

Comment:
hmm .. I don't think we should be using expandable sections here. That's User Guide - it's purpose is to be more comprehensive, right?

Reply:
Hmmm we still have some expanding sections for implementation details. But yeah I expanded this and it looks good.